.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-arch: ['amd64', 'arm', 'arm64']
+        go-arch: ['amd64', 'arm']
```

Comment:
I think this matrix just needs to change unfortunately. Instead of cross compiling, compile using a native machine. Didn't have time to check yesterday if `mac-os` runner actually has `arm64` arch.

Reply:
No linux arm support apparently: https://github.com/actions/runner-images/issues/5631
Similarly, mac-os doesn't seem to be using the new M1 arm64 chip: https://github.com/github/roadmap/issues/528

Reply:
Update: can build on arm64, wasn't using cross compiler correctly. Can't on arm32 since [wasmvm can't support it](https://github.com/CosmWasm/wasmvm#builds-of-libwasmvm) due to Wasmer not supporting it.

The previous does apply if we'd like to run the tests on arm64 via github runners tho.